applications/luci-app-ledtrig-ath10k/htdocs/luci-static/resources/view/system/led-trigger/ath10k.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+'require baseclass';
+'require form';
+'require fs';
+'require uci';
+
+return baseclass.extend({
+	trigger: _('Wirless trigger (ath10k)'),
```

Comment:
There's a typo in the caption.

Reply:
@jow- thanks done.

For this to work, however, this [dependency](https://patchwork.ozlabs.org/project/openwrt/list/?series=290888) must first be merged into the openwrt base.

Reply:
ping

Reply:
@systemcrash Thanks for the hint.
I have already update this see my push --force 2 years ago.
So I will resolve this.
@jow- @systemcrash but I am still waiting for the merge of the pullrequest this change depends on.